plugins/keycloak-backend/src/providers/KeycloakOrgEntityProvider.ts

```diff
@@ -269,7 +269,7 @@ export class KeycloakOrgEntityProvider implements EntityProvider {
 }
```

Comment:
You missed the `Logger` usage in https://github.com/janus-idp/backstage-plugins/blob/835e63f3995fb840108118681f6e012f7ac88f08/plugins/keycloak-backend/src/providers/KeycloakOrgEntityProvider.ts#L183